modules/doc/content/newsletter/2024/2024_07.md

```diff
@@ -58,6 +58,24 @@ for a complete description of all MOOSE changes.
 - `System::is_initialized()` is now `const`
 - Added `LibmeshPetscCall` macro that can wrap any PETSc function call without need for `LIBMESH_CHKERR(ierr)`
 
+### `2024.07.27` Update
+
+- Bounding box testing for .msh $Entities in GmshIO::read(), to detect
```

Comment:
If for some reason you have to push again, I might suggest moving the `LibmeshPetscCall` item down to this new update

Reply:
Yeah; I'm still not sure how I got the news item in the previous update but not the actual commit.  I usually just `git log --merges hash..` and expand+reorder what I read there.  I wonder if I somehow managed to `git checkout origin/master` right before the `LibmeshPetscCall` merge and `git log --merges hash..origin/master` right after.

Reply:
It's because I suggested that you add it to the last newsletter entry. So it was my flub